pom.xml

```diff
@@ -343,11 +333,13 @@
             <configuration>
               <failOnWarning>true</failOnWarning>
               <ignoredDependencies>
+                <ignoredDependency>log4j:log4j:jar:*</ignoredDependency>
```

Comment:
log4j is used in test code to change log levels.  however, it is brought in by slf4-logj412 so it does not need to explicitly declared.